tests/test_backend_consistency.py

```diff
@@ -115,7 +115,12 @@ def test_hypotest_q_mu(
         pyhf.set_backend(backend)
 
         q_mu = pyhf.infer.test_statistics.qmu(
-            1.0, data, pdf, pdf.config.suggested_init(), pdf.config.suggested_bounds(),
+            1.0,
+            data,
+            pdf,
+            pdf.config.suggested_init(),
+            pdf.config.suggested_bounds(),
+            None,
```

Comment:
There should be a test that actually tests the use of `fixed_vals` and not just the `None` case.

Reply:
Included in the test, see https://github.com/scikit-hep/pyhf/pull/846#issuecomment-624550955